test/spec.yml

```diff
@@ -457,6 +457,47 @@
         exit: 1
         err: "Empty string env 'EMPTY' in binaris.yml is not supported"
 
+- test: "concurrent execution model: deploy invoke multiple times"
+  steps:
+    -   in: |-
+            cat <<EOF >/home/dockeruser/test/binaris.yml
+            functions:
+              concurrentfunc:
+                file: function.js
+                entrypoint: handler
+                runtime: node8
+                executionModel: concurrent
+            EOF
+    -   in: |-
+            cat <<EOF >/home/dockeruser/test/function.js
+            function msleep(ms) {
+              return new Promise((resolve) => {
+                  setTimeout(resolve, ms);
+              });
+            }
+            let duringFlight = false;
+            exports.handler = async () => {
+              if (duringFlight) {
+                return true;
+              } else {
+                duringFlight = true;
+                await msleep(10000);
+                duringFlight = false;
+                return false;
+              }
+            }
+            EOF
+    -   in: bn deploy concurrentfunc
+        out: |-
+            Deployed function concurrentfunc
+            Invoke with one of:
+              "bn invoke concurrentfunc"
+              "curl -H X-Binaris-Api-Key:* https://*"
+    -   in: "(for i in `seq 1 12`; do bn invoke concurrentfunc & done) | grep true"
```

Comment:
Why 12?

Reply:
we were a bit tired when deciding on 12, we actually need `minBolts+1` so in the `testing` realm, that would be 3